jit/src/lib.rs

```diff
@@ -33,10 +34,19 @@ struct Jit {
     module: JITModule,
 }
 
+// TODO: print should return None
+pub fn print_fun(a: i64) -> i64 {
+    println!("PRINT: {a}");
```

Comment:
I am confused  here. would `print(1)` be compiled to print `PRINT: {a}` instead of `{a}`? Then it will not be compatible to pure python behavior.